packages/anoncreds/package.json

```diff
@@ -31,6 +31,8 @@
   },
   "devDependencies": {
     "@aries-framework/node": "0.3.3",
+    "@hyperledger/indy-vdr-nodejs": "^0.1.0-dev.6",
```

Comment:
anoncreds doesn't need to depend on indyVdr I think? I think you can import `indyVdrModuleConfig` from the indy-vdr package test folder